Riot/Model/Room/RoomDataSource.m

```diff
@@ -148,7 +149,7 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
             // Some vertical whitespaces are added in message text view (see RoomBubbleCellData class) to insert correctly multiple receipts.
             bubbleCell.bubbleOverlayContainer.backgroundColor = [UIColor clearColor];
             bubbleCell.bubbleOverlayContainer.alpha = 1;
-            bubbleCell.bubbleOverlayContainer.userInteractionEnabled = NO;
+            bubbleCell.bubbleOverlayContainer.userInteractionEnabled = YES;
```

Comment:
FYI, I agree we have to enable the user interaction here on bubble overlay to catch the tap on RR container.
Unfortunately this change will break the message selection in the room history for the bubbles with RR. Don't worry about this, I will take this in charge.